examples/js/postprocessing/EffectComposer.js

```diff
@@ -127,12 +132,8 @@ THREE.EffectComposer.prototype = {
 
 	setSize: function ( width, height ) {
```

Comment:
Previously done here: https://github.com/mrdoob/three.js/pull/6820/files#diff-f69be6f1b9bed3c3d7387ed929cd29faR113 however this function is now defunct.